core/src/components/popover-canvas/popover-canvas.stories.tsx

```diff
@@ -70,22 +70,22 @@ const ComponentPopoverCanvas = ({ canvasPosition }) => {
 
       <!-- The 'referenceEl' prop can be used instead of 'selector', 
       wich might be preferable in frameworks like React -->
-      <sdds-popover-canvas
+      <tds-popover-canvas
         placement="${canvasPosLookup[canvasPosition]}"
         selector="#trigger"
-        class="sdds-u-p2">
+        class="tds-u-p2">
         <h2>A Popover Canvas!</h2>
         <p>
           Where you can put anything you want!
         </p>
         <p>
           <a target="_blank" rel="noopener noreferrer" href="https://digitaldesign.scania.com">Even links!</a>
         </p>
-      </sdds-popover-canvas>
+      </tds-popover-canvas>
 
       <!-- demo-wrapper code below is for demonstration purposes only -->
       <div class="demo-wrapper">
-        <span class="sdds-u-mr2">Click icon for Popover Canvas</span>
+        <span class="tds-u-mr2">Click icon for Popover Canvas</span>
         
         <sdds-button aria-label="menu" onlyIcon id="trigger" type="ghost" size="sm">
           <tds-icon slot="icon" class="sdds-btn-icon" size="16px" name="kebab"></tds-icon>
```

Comment:
```suggestion
          <tds-icon slot="icon" size="16px" name="kebab"></tds-icon>
```
I think the class is not anymore available as we removed it with CSS native removal. Please double-check if it looks OK or if some class is needed after all.